tracer/build/_build/Build.VariableGenerations.cs

```diff
@@ -1782,13 +1782,13 @@ void GenerateMacosDotnetToolNugetSmokeTestsMatrix()
                 {
                     var images = new SmokeTestImage[]
                     {
-                        // macos-11/12 environments are no longer available in Azure Devops
-                        new (publishFramework: TargetFramework.NETCOREAPP3_1, "macos-13", "macos", "13"),
-                        new (publishFramework: TargetFramework.NET5_0, "macos-13", "macos", "13"),
-                        new (publishFramework: TargetFramework.NET6_0, "macos-13", "macos", "13"),
-                        new (publishFramework: TargetFramework.NET7_0, "macos-13", "macos", "13"),
-                        new (publishFramework: TargetFramework.NET8_0, "macos-13", "macos", "13"),
-                        new (publishFramework: TargetFramework.NET9_0, "macos-13", "macos", "13"),
+                        // macos-11/12/13 environments are no longer available in Azure Devops
```

Comment:
Ahh some of these actually appear to be the same so we can remove them, but they aren't all the same / dupes